lib/generators/geoblacklight/templates/settings.yml

```diff
@@ -216,6 +216,13 @@ RELATIONSHIPS_SHOWN:
     label: geoblacklight.relations.version_of_descendants
     query_type: descendants
 
+# Vector data download formats to be displayed
+DOWNLOAD_FORMATS:
+  VECTOR:
+    - 'Shapefile'
+    - 'KMZ'
+    - 'GeoJSON'
```

Comment:
I don't see a problem with making CSV download a standard option. I like that the download formats are configurable, but adding CSV to this list make it's easy to test and develop. Better not to hide features by default.

Reply:
Will do!